spec/classes/rabbitmq_spec.rb

```diff
@@ -148,8 +148,15 @@
           end
 
           if facts[:systemd]
+            selinux_ignore_defaults = if facts[:os]['family'] == 'RedHat'
+              true
+            else
+              false
+            end
```

Comment:
This can be simplified:
```suggestion
            selinux_ignore_defaults = facts[:os]['family'] == 'RedHat'
```